mill

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env sh
+
+# This is a wrapper script, that automatically download mill from GitHub release pages
+# You can give the required mill version with --mill-version parameter
+# If no version is given, it falls back to the value of DEFAULT_MILL_VERSION
+
+DEFAULT_MILL_VERSION=0.2.7-10-db8525
+
+set -e
+
+if [ "x$1" = "x--mill-version" ] ; then
+  shift
+  if [ "x$1" != "x" ] ; then
+    MILL_VERSION="$1"
+    shift
+  else
+    echo "You specified -v without a version."
```

Comment:
Should this be `"You specified --mill-version without a version."`

Reply:
You are right, I changed it from a simple `-v` to `--mill-version` to avoid prospective collisions.